blog/posts/_data.json

```diff
@@ -0,0 +1,20 @@
+{
+  "post-2": {
+    "title": "Sample Post 2",
+    "date": "January 4, 2015",
+    "authors": "GothamSass Author2",
+    "description": "Sed elementum consectetur sem, lobortis viverra dolor tincidunt eu. Pellentesque nec malesuada tortor, nec rutrum est. Donec fermentum ut nunc at scelerisque. Morbi elementum, lacus at bibendum vehicula, neque felis semper urna, at auctor mi lorem nec ex."
+  },
+  "post-1": {
+    "title": "Sample Post 1",
+    "date": "January 3, 2015",
+    "authors": "GothamSass Author1",
```

Comment:
If we are allowing multiple authors, which I believe we should, then this needs to become an array. Additionally, I think we should expand the author section a bit so we can have an _About the Author_ section.

Here's a proposal for new author object

````json
"authors": [
  {
     "name": "Awesome author",
     "bio": "Awesome author is very awesome and does awesome things with code and design.",
     "email": "awesome@coolkid.com",
     "website": "coolkid.com",
     "social": {
        "github": "awesomely",
        "twitter": "awesomely" 
      }
  }
]
````
* Authors should be free to enter `null` for any info they don't want to give. 
* Social is an array that can accept any key/value pair so authors can list whatever they want, i.e. dribbble, instagram, soundcloud etc etc